File: pset1/cash.c
#include <stdio.h>
#include <cs50.h>
#include <math.h>

int main(void)
{
    float change;
    do
    {
        change = get_float("Change owed: ");
    }
    while (change < 0);
    printf("Change owed: %.2f\n", change);

    //convert dollars to cents (mult by 100 then round off)
    int cents = round(change * 100);
    printf("%i",cents);
    int count = 0;

    //Subtract coins starting with highest value until no more can be subtracted
    while (cents >= 25)
    {
        cents = cents - 25;
        count++;
    }
    while (cents >= 10)
    {
        cents = cents - 10;
        count++;
    }
        while (cents >= 5)
    {
        cents = cents - 5;
        count++;
    }
        while (cents >= 1)
    {
        cents = cents - 1;
        count++;
    }
    //print count of number coins used
    printf ("Count: %i\n",count);

}

File: pset6/cash/cash.py
from cs50 import get_float

#include <math.h>

while True:
    change = get_float("Change owed: ")
    if change > 0:
        break
print(f"Change owed: {change}")

#convert dollars to cents
cents = round(change * 100)
print(f"{cents}")
count = 0

count += cents // 25
count += (cents % 25) // 10
count += ((cents % 25) % 10) // 5
count += ((cents % 25) % 10) % 5

print(f"Count: {count}")
